1-Pflichtenheft/pflichtenheft.tex

```diff
@@ -168,20 +172,21 @@ \section{Funktionale Anforderungen}
 \fulfills{crt:create}
 \fulfills{crt:gui-es}
 \fulfills{crt:certdist}
-Der \gls{Wahlleiter} fügt die \gls{Waehler} mit ihrem Name hinzu. Das System generiert automatisch die erforderlichen \glspl{Zertifikat}{Zertifikate}. Die \glspl{Zertifikat}{Zertifikate} werden im Netzwerk verteilt. Nur die hinzugefügten \gls{Waehler} sind zur Teilname berechtigt. \\
+Der \gls{Wahlleiter} fügt die \gls{Waehler} mit ihrem Namen hinzu. Das System generiert automatisch die erforderlichen \glspl{Zertifikat}. Die \glspl{Zertifikat} werden im Netzwerk verteilt. Nur die hinzugefügten \gls{Waehler} sind zur Teilname berechtigt. \\
 Der \gls{Wahlleiter} kann außerdem die \glspl{Zertifikat} per Email an die \gls{Waehler} senden. % Wie vermerkt man hier das das Teil des Kannkriteriums ist?
 
 \functionality{Hinzufügen von Kandidaten}{fnc:candidate-add}
 \fulfills{crt:create}
 \fulfills{crt:gui-es}
-Der \gls{Wahlleiter} fügt die \glslink{Kandidat}{Kandidaten} mit ihrem Name hinzu. Es ist außerdem möglich jedem \glslink{Kandidat}{Kandidaten} eine Beschreibung zu geben, die dem Wähler in ihrer \glslink{Benutzeroberflaeche}{GUI} angezeigt wird. Das System propagiert die \glslink{Kandidat}{Kandidaten} automatisch an das Netzwerk
+Der \gls{Wahlleiter} fügt die \glslink{Kandidat}{Kandidaten} mit ihrem Namen hinzu. Es ist außerdem möglich jedem \glslink{Kandidat}{Kandidaten} eine Beschreibung zu geben, die den \glslink{Waehler}{Wählern} in ihrer \glslink{Benutzeroberflaeche}{GUI} angezeigt wird. Das System propagiert die \glslink{Kandidat}{Kandidaten} automatisch an das Netzwerk
 
 \functionality{Aktivierung der Wahl}{fnc:activate}
 \fulfills{crt:create}
 \fulfills{crt:gui-es}
 Der \gls{Wahlleiter} bestätigt seine Einstellungen zur Wahl. Mit der Bestätigung beginnt die Übertragung der Informationen in das Netzwerk. Die \gls{Wahl} startet zum festgelegten Zeitpunkt.
 
 \functionality{Wahlfunktion für \gls{Waehler}}{fnc:vote}
+%Glossary links entweder in allen Überschriften oder keinen
```

Comment:
In keinen, außer der Begriff kommt sonst nirgendwo vor, ist hier aber nicht der Fall.

Reply:
Alles klar, Glossar verlinkungen sollten sowieso noch vereinheitlicht werden, kümmer ich mich dann morgen drum.